ls1intum/Artemis
# `General`: Update exercise header on new results

### Checklist
#### General
#### Client

### Motivation and Context
When a new results gets reported via the websocket, the exercise header did not update the received points:

![grafik](https://user-images.githubusercontent.com/26540346/203954325-a1c36178-d44e-4126-8337-05fa8342c3b3.png)


### Description
The `gradedStudentParticipation` attribute was not updated correctly.

### Steps for Testing

Prerequisites:
- 1 Student
- 1 Programming Exercise

1. Open the PE detail view
2. Push a new submission via the local IDE
3. Wait for the build to finish
4. Check that the points in the header get updated to the new result

### Review Progress

#### Code Review
#### Manual Tests

### Test Coverage
